packages/vvisp/test/scripts/console.test.js

```diff
@@ -0,0 +1,349 @@
+const rewire = require('rewire');
+const chai = require('chai');
+const path = require('path');
+const chalk = require('chalk');
```

Comment:
Never used, remove plz!

Reply:
only `const path = require('path');` is not used!

Reply:
I removed const chalk = require('chalk');